src/com/core/tictactoe/Board.java

```diff
@@ -75,6 +75,24 @@ String winnerSign() {
         return freePlaces;
     }
 
+    String getActivePlayerSign() {
```

Comment:
There could be a more concise way to calculate the current marker if you base it on the total number of marks on the board...

Reply:
AWESOME idea!

Reply:
Done.